examples/eks-cluster-with-observability/README.md

```diff
@@ -0,0 +1,215 @@
+# EKS Cluster with Amazon Managed Grafana and Amazon OpenSearch Service
+
+This example demonstrates how to use the AWS EKS Accelerator for Terraform to deploy a new Amazon EKS Cluster with Prometheus for metrics and Fluent Bit for logging. It also provisions Amazon Managed Prometheus (AMP), Amazon OpenSearch Service within a VPC, and integrates AMP with Amazon Managed Grafana (AMG). It also deploys an EC2 instance to let us test OpenSearch. Lastly, it includes a sample workload, provisioned with ArgoCD, to generate logs and metrics.
```

Comment:
This example demonstrates how to use the Amazon EKS Accelerator for Terraform to deploy a new Amazon EKS Cluster with Prometheus server for metrics and AWS Fluent Bit for logs.

Reply:
I has been asked not to refer **AMP** or **AMG** in any docs or blogs by the Amazon Observability team.

Replace AMP with Amazon Managed Prometheus and AMG with Amazon Managed Grafana